src/overlays/actors/ovl_Bg_Spot18_Basket/z_bg_spot18_basket.c

```diff
@@ -21,39 +36,397 @@ const ActorInit Bg_Spot18_Basket_InitVars = {
+        { 0, { { 0, 2040, 0 }, 54 }, 100 },
+    },
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, 0x00, 0x01, 0x00 },
+        { 1, { { 0, 1400, 0 }, 13 }, 100 },
+    },
+};
+
+static ColliderJntSphInit sJntSphInit = {
+    { COLTYPE_UNK10, 0x00, 0x09, 0x09, 0x20, COLSHAPE_JNTSPH },
+    2,
+    sJntSphItemsInit,
+};
+
+static s16 D_808B85C8[] = { 0x8000, 0x2AAA, 0xD555, 0x0000 };
+
+void func_808B7710(Actor* thisx, GlobalContext* globalCtx) {
+    BgSpot18Basket* this = THIS;
+
+    Collider_InitJntSph(globalCtx, &this->colliderJntSph);
+    Collider_SetJntSph(globalCtx, &this->colliderJntSph, &this->dyna.actor, &sJntSphInit, &this->colliderJntSphItems);
+    this->dyna.actor.colChkInfo.mass = 0xFF;
+}
+
+void func_808B7770(BgSpot18Basket* this, GlobalContext* globalCtx, f32 arg2) {
+    Vec3f acceleration;
+    Vec3f velocity;
+    Vec3f position;
+    f32 cosValue;
+    s32 i = 0;
+    f32 randomValue;
+    static s16 D_808B85D0 = 0;
+    f32 sinValue;
+    s32 numberOfLoops = 2;
+
+    while (i != numberOfLoops) {
+        if (globalCtx) {}
+        if (!(arg2 < Math_Rand_ZeroOne())) {
+            D_808B85D0 += 0x7530;
+
+            sinValue = Math_Sins(D_808B85D0);
+            cosValue = Math_Coss(D_808B85D0);
+
+            randomValue = (Math_Rand_ZeroOne() * 35.0f) + 35.0f;
+
+            position.x = (randomValue * sinValue) + this->dyna.actor.posRot.pos.x;
+            position.y = this->dyna.actor.posRot.pos.y + 10.0f;
+            position.z = (randomValue * cosValue) + this->dyna.actor.posRot.pos.z;
+
+            velocity.x = sinValue;
+            velocity.y = 0.0f;
+            velocity.z = cosValue;
+
+            acceleration.x = 0.0f;
+            acceleration.y = 0.5f;
+            acceleration.z = 0.0f;
+
+            func_800286CC(globalCtx, &position, &velocity, &acceleration, ((Math_Rand_ZeroOne() * 16) + 80),
+                          ((Math_Rand_ZeroOne() * 30) + 80));
+        }
+        i++;
+    }
+}
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneForward, 1000, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 500, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 1000, ICHAIN_STOP),
+};
+
+static s16 D_808B85E4[] = { 0xF060, 0x0320, 0x0FA0 };
+
+void BgSpot18Basket_Init(Actor* thisx, GlobalContext* globalCtx) {
+    BgSpot18Basket* this = THIS;
+    Actor* actor = &this->dyna.actor;
+    s32 temp = 0;
+
+    DynaPolyInfo_SetActorMove(&this->dyna, DPM_UNK3);
+    func_808B7710(this, globalCtx);
+    DynaPolyInfo_Alloc(&D_06002154, &temp);
+
+    this->dyna.dynaPolyId = DynaPolyInfo_RegisterActor(globalCtx, &globalCtx->colCtx.dyna, actor, temp);
+
+    Actor_ProcessInitChain(actor, sInitChain);
+    ActorShape_Init(&actor->shape, 0.0f, ActorShadow_DrawFunc_Circle, 15.0f);
+    actor->initPosRot.pos.y = actor->initPosRot.pos.y + 0.01f;
+    actor->posRot.pos.y = actor->initPosRot.pos.y;
```

Comment:
```suggestion
    actor->posRot.pos.y = actor->initPosRot.pos.y = actor->initPosRot.pos.y + 0.01f;
```

Reply:
This actually doesn't match but I changed the first line to be +=.